operator/src/main/java/org/keycloak/operator/v2alpha1/KeycloakRealmImportJob.java

```diff
@@ -55,7 +56,9 @@ public KeycloakRealmImportJob(KubernetesClient client, KeycloakRealmImport realm
 
     @Override
     protected Optional<HasMetadata> getReconciledResource() {
-        if (existingJob == null) {
+        if (existingDeployment == null) {
+            throw new ResourceNotFoundException("Keycloak Deployment not found: " + getKeycloakName());
```

Comment:
This is an unrelated enhancement. I ran into unspecified NPE when fixing the RealmImport test, this is to make the dev experience better in the future.

Re-using a Fabric8 exception as it represents a missing K8s resource.

Reply:
👍